app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self, distance_from_city_center: int,
+            clean_power: int, average_rating: int,
+            count_of_ratings: int
+    ) -> None:
```

Comment:
Code Style Issue: The variable name `count_of_ratings` is not descriptive enough. It's better to use `rating_count` as it is more conventional and easier to read.